Makefile

```diff
@@ -154,7 +154,8 @@ all-tests: eunit test
 system-test:
 	@./rebar3 as system_test do ct --dir system_test --logdir system_test/logs --config system_test/cfg $(CT_TEST_FLAGS)
 
-system-test-deps: | system_test/aest_hard_fork_SUITE_data/db/mnesia_ae-uat-epoch_backup.tar ;
+system-test-deps: | system_test/aest_hard_fork_SUITE_data/db/mnesia_ae-uat-epoch_backup.tar
+	docker pull aeternity/epoch:v0.11.0
```

Comment:
What if we do also `docker tag aeternity/epoch:v0.11.0 aeternity/epoch:local`? It would make `:local` alias of `:v0.11.0` so then other changes in the code are not necessary.

Reply:
Because other tests may expect :local to be local - I am thinking of sync tests https://github.com/aeternity/epoch/blob/a9798667c3a7e49926ae4caa874d7625b2b1e58f/system_test/aest_sync_SUITE.erl#L56

Reply:
Maybe you could use different tag for the fork tests? I like the @jur0 idea.